src/content/docs/en/recipes/dynamically-importing-images.mdx

```diff
@@ -0,0 +1,146 @@
+---
+title: Dynamically Import Images
+description: Learn how to dynamically import images using Vite's import.meta.glob function
+i18nReady: true
+type: recipe
+---
+import FileTree from '~/components/FileTree.astro'
+
+Local [images](/en/guides/images/) must be imported into `.astro` files in order to display them. There will be times where you will want or need to dynamically import the image paths of your images instead of explicitly importing each individual image.
+
+In this recipe, you will learn how to dynamically import your images using Vite's `import.meta.glob` function. You will build a card component that displays the name, age, and photo of a person.
+
+## Recipe
+
+1. Create a new `assets` folder under the `src` directory and add your images inside that new folder. 
+
+<FileTree>
+- src/
+  - assets/
+    - avatar-1.jpg
+    - avatar-2.png
+    - avatar-3.jpeg
+</FileTree>
+
+:::note
+`assets` is a popular folder name convention for placing images but you are free to name the folder whatever you like.
+::: 
+
+2. Create a new Astro component for your card and import the `<Image />` component.
+
+    ```astro title="src/components/MyCustomCardComponent.astro" 
+    ---
+     import { Image } from 'astro:assets';
+    ---
+    ```
+
+3. Specify the `props` that your component will receive in order to display the necessary information on each card. You can optionally define their types, if you are using TypeScript in your project.
+
+    ```astro title="src/components/MyCustomCardComponent.astro" ins={4-11}
+    ---
+     import { Image } from 'astro:assets';
+
+     interface Props {
+        imagePath: string;
+        altText: string;
+        name: string;
+        age: number;
+     }
+
+     const { imagePath, altText, name, age } = Astro.props;
+    ---
+    ``` 
+
+4. Create a new `images` variable and use the `import.meta.glob` function which returns an object of all of the image paths inside the `assets` folder. 
+
+    ```astro title="src/components/MyCustomCardComponent.astro" ins={12}
+    ---
+     import { Image } from 'astro:assets';
+
+     interface Props {
+        imagePath: string;
+        altText: string;
+        name: string;
+        age: number;
+     }
+     
+     const { imagePath, altText, name, age } = Astro.props;
+     const images = import.meta.glob('/src/**/*.{jpeg,jpg,png,gif}')
+    ---
+    ```
+
+5. Use the props to create the markup for your card component. 
+
+    ```astro title="src/components/MyCustomCardComponent.astro" ins={14-18}
+    ---
+     import { Image } from 'astro:assets';
+
+     interface Props {
+        imagePath: string;
+        altText: string;
+        name: string;
+        age: number;
+     }
+     
+     const { imagePath, altText, name, age } = Astro.props;
+     const images = import.meta.glob('/src/**/*.{jpeg,jpg,png,gif}')
```

Comment:
Based on other comments, I think we can add a type comment for ts users.
```suggestion
     \\ Type: Record<string, () => Promise<{default: ImageMetadata}>>
     const images = import.meta.glob('/src/**/*.{jpeg,jpg,png,gif}')
```

Reply:
Wrong slashes, you need `//`

Reply:
I'll amend for the commit!  🙌